src/plugins/equalheight/reports/1-acr-wcag21.json

```diff
@@ -0,0 +1,494 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json-ld",
+
+  "@type": "acr:ConformanceReport",
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/wet-boew/demos/equalheight/equalheight-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Only on the enhanced and basic HTML version of the add to euqal height component.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ".wb-euqalheight"
+    }
+  },
+
+  "earl:assertedBy": {
+    "@id": "_:acrAssertor",
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Hongbin Yu (Github: @hongbinyu413)",
+      "foaf:homepage": "https://github.com/hongbinyu413",
+      "@type": ["earl:Assertor", "foaf:Person"]
+    }
+  },
+
+  "dct:created": "2022-09-24",
+  "dct:issued": "2023-03-02",
+  "dct:modified": "2023-06-19",
```

Comment:
Is the date are accurate? The creation date should be equal or greater than of the assessment date. And I do think all date should be between March 2023 to July 2023